src/main/scala/diplomacy/Nodes.scala

```diff
@@ -253,6 +265,27 @@ sealed abstract class MixedNode[DI, UI, EI, BI <: Data, DO, UO, EO, BO <: Data](
   val inward = this
   val outward = this
 
+  /** debug info of nodes binding. */
+  def bindingInfo: String =
+    s"""${oBindings.size} outward nodes binded: [${oBindings.map(n => s"${n._3.serialize}-${n._2.name}").mkString(",")}]
+       |${iBindings.size} inward nodes binded: [${iBindings.map(n => s"${n._3.serialize}-${n._2.name}").mkString(",")}]
```

Comment:
Should be "outward nodes bound" instead of "binded" ... sorry english is awful! 😅

Reply:
Yep, so sorry for this, trying hard to improve these days.